provisioner/ansible.cfg

```diff
@@ -10,7 +10,8 @@
 [defaults]
 interpreter_python = auto_silent
 stdout_callback = yaml
-inventory      = hosts
+inventory      = ./craigbsec/instructor_inventory.txt
```

Comment:
you are going to want to gitignore this file so you don't change the public one @craig-br